cf-check/diagnose.c

```diff
@@ -534,8 +551,32 @@ size_t diagnose_files(
     for (int i = 0; i < length; ++i)
     {
         const char *filename = SeqAt(filenames, i);
+        const char *symlink;
         int r;
-        if (foreground)
+        char *symlink_target = follow_symlink(filename);
+        bool broken_symlink_handled = false;
+        if (symlink_target != NULL)
+        {
+            symlink = filename;
+            // If the LMDB file path is a symlink
+            filename = symlink_target;
+            struct stat sb;
+            if (stat(symlink_target, &sb) != 0 && errno == ENOENT)
```

Comment:
You can use `access(symlink_target, F_OK)`.

Reply:
Fixed, it's nice to not have to use a stat buffer, but I also feel that `access` is not a great name, when  all I want to do is check if a file _exists_. Probably an inline function `FileExists` or something could be added, but it doesn't matter that much.